.travis.yml

```diff
@@ -1,29 +0,0 @@
-language: php
```

Comment:
@stefanius Other repo's use the .travis.yml, here it is replaced by travis.yml?

Reply:
It should be .travis.yml. I can't see why its chaged?